index.html

```diff
@@ -5,7 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta property="og:title" content="Filmster">
   <meta property="og:description" content="A web application for movie enthusiats who want to look up detailed information about a movie.">
-  <meta property="og:image" content="images/logo.ico">
+  <meta property="og:image" content="https://images.pond5.com/yellow-computer-monitor-screen-icon-footage-147692385_iconl.jpeg">
```

Comment:
I recommend downloading the file to your project. Then use the absolute url.

```html
  <meta property="og:image" content="https://DevJosephLee.github.io/Filmster/images/logo.jpeg">

```

Reply:
I decided to use the app favicon as the preview!